test/framework/src/main/java/org/elasticsearch/search/aggregations/AggregatorTestCase.java

```diff
@@ -524,6 +560,163 @@ protected static IndexReader maybeWrapReaderEs(DirectoryReader reader) throws IO
         }
     }
 
+    protected AggregationBuilder createAggBuilderForTypeTest(MappedFieldType fieldType, String fieldName) {
+        return null;
```

Comment:
Maybe throw here by default instead of returning `null`?  Otherwise we're going to get an NPE deep in the aggregation code when someone forgets to implement this, and it won't be at all clear why.